candig/ingest/ingest.py

```diff
@@ -490,22 +490,31 @@ def main():
                             records = [records]
 
                         for record in records:
-                            local_id_list = []
-                            for x in metadata_map[metadata_key][table]['local_id']:
-                                if record.get(x):
-                                    local_id_list.append(record[x])
-                                else:
-                                    print("Skipped: Missing 1 or more primary identifiers for record in: {0} needs {1}, received {2}".format(
-                                        table,
-                                        metadata_map[metadata_key][table]['local_id'],
-                                        local_id_list,
-                                        ))
-                                    local_id_list = None
-                                    break
-                            if not local_id_list:
-                                continue
-
-                            local_id = "_".join(local_id_list)
+                            localId = ''
```

Comment:
Variable `localId` is never used anywhere so this line can simply be deleted

Reply:
Thanks!